books/kestrel/file-io-light/top.lisp

```diff
@@ -77,4 +77,8 @@
 (include-book "file-length-dollar")
 (include-book "file-is-newer-thanp")
 
+;; Books grouped together for a specific purpose:
+(include-book "output-theory")
+(include-book "output-theory-test")
```

Comment:
I wouldn't include test files into top files. The main purpose of top files is to ensure the absence of name conflicts, but that doesn't matter for tests.

Reply:
Done, thanks!